src/ui/public/state_management/__tests__/state.js

```diff
@@ -272,17 +268,18 @@ describe('State Management', () => {
 
       describe('error handling', () => {
         it('notifies the user when a hash value does not map to a stored value', () => {
-          const { state, notifier } = setup({ storeInHash: true });
+          toastNotifications.remove(toastNotifications.list[0]);
```

Comment:
I'm confused why the first step in this test would be to delete the notification at position `0`... Why not clear notifications after each test?